### PR TITLE
Create the project and about us sections

### PR DESCRIPTION
**Pull request by Jolak5**
I created the about us section and the card

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,5 +53,124 @@ <h1>
...
+    <div class="top-tile">
+      <h1 class="card-title">
+        My Recent <br />
+        Works
+      </h1>
+      <hr />
+    </div>
+    <!-- The main card container -->
+    <div class="card-container">
+      <div class="card">
```
Comment (ghost) [inline index.html]: - [ ] Kindly wrap your work container in a `<section>` tag for accessibility reasons

File: index.html [hunk 2]
```diff
@@ -52,5 +53,124 @@ <h1>
...
+          <h4>Multi-Post Stories <br />Gain+Glory</h4>
+          <ul>
+            <div class="top-card"></div>
+            <li>Html</li>
+            <li>Css</li>
+            <li>Javascript</li>
+          </ul>
+          <button class="card-button">See Projects</button>
+        </div>
+      </div>
```
Comment (ghost) [inline index.html]: - [ ] Kindly add a margin between the language skills in your cards to match the original design

image

### Checkout Figma

image

- [ ] Also kindly adjust the border-color of your buttons to match the Figma design

image

### Checkout Figma

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @Jolak5 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Jolak5) [reply index.html]: Thank you for your review. As regards the border-color, I copied the style directly from figma. But I will adjust it

**Review by levy002 — APPROVED** [review]
## Hello @Jolak5 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**